xlml/utils/metric_test.py

```diff
@@ -315,27 +316,116 @@ def test_add_airflow_metadata(self):
 
           self.assert_metric_and_dimension_equal([], [], actual_value, expected_value)
 
+  def test_add_test_config_metadata(self):
+    base_id = "test_run"
+    uuid = hashlib.sha256(str(base_id + str(0)).encode("utf-8")).hexdigest()
+
+    raw_meta = [
+        [
+            bigquery.MetadataHistoryRow(
+                job_uuid=uuid,
+                metadata_key="framework",
+                metadata_value="jax",
+            )
+        ]
+    ]
+    task_test_config = test_config.TpuVmTest(
+        test_config.Tpu(
+            version=TpuVersion.V4,
+            cores=8,
+            runtime_version=RuntimeVersion.TPU_UBUNTU2204_BASE.value,
+            network="default",
+            subnetwork="default",
+        ),
+        test_name="test_name",
+        set_up_cmds="set_up_cmds",
+        run_model_cmds="run_model_cmds",
+        time_out_in_min=60,
+        task_owner="test_owner",
+        num_slices=1,
+    )
+
+    task_gcp_config = gcp_config.GCPConfig(
+        project_name="test_project",
+        zone="tpu_zone",
+        dataset_name="dataset_name",
+        dataset_project="test_project",
+        composer_project="test_project",
+    )
+
+    actual_value = metric.add_test_config_metadata(
+        base_id,
+        task_test_config,
+        task_gcp_config,
+        raw_meta,
+    )
+    print("actual_value", actual_value)
```

Comment:
Do we need these prints?

Reply:
I followed the same pattern as the existing tests https://github.com/GoogleCloudPlatform/ml-auto-solutions/blob/5e0d91f69f6a68640d3d8be6ffa486e961d3fd7b/xlml/utils/metric_test.py#L285-L293. I think it would be good to have the prints in case the test fails we could see the difference.